src/sorunlib/stimulator.py

```diff
@@ -70,20 +70,29 @@ def calibrate_tau(duration_step=20,
     filter_disable : bool, optional
         If True, will disable the downsample filter before streaming. Defaults to True.
     filter_order : int, optional
-        Order of the downsample filter for SMuRF. Defaults to 4.
+        Order of the downsample filter for SMuRF. Defaults to None.
```

Comment:
But what does None cause the order to be?  Does pysmurf default to 4 in that case?  Clearer to say "If None is passed, the pysmurf default of 4 will be used" (or something like that).